components/gitpod-db/go/encryption_test.go

```diff
@@ -2,25 +2,26 @@
 // Licensed under the GNU Affero General Public License (AGPL).
 // See License.AGPL.txt in the project root for license information.
 
-package db
+package db_test
```

Comment:
To test the encryption functions from "the outside" as a consumer of the package.